File: backend/src/main/java/com/pedro256/sga/dto/app/user/UserDto.java
package com.pedro256.sga.dto.app.user;

import lombok.Data;

import java.util.UUID;

@Data
public class UserDto {
    private UUID id;
    private String name;
    private String secondName;
    private String email;
    private String CPF;
    private String phone;
    private String keycloakId;
}

File: frontend/src/dto/user/UserDto.ts
export class UserDto {
    id: string;
    name: string;
    secondName: string;
    email: string;
    CPF: string;
    phone: string;
    keycloakId: string;

    constructor(id: string, name: string, secondName: string, email: string, CPF: string, phone: string, keycloakId: string) {
        this.id = id;
        this.name = name;
        this.secondName = secondName;
        this.email = email;
        this.CPF = CPF;
        this.phone = phone;
        this.keycloakId = keycloakId;
    }
}
